### DISCUSSION TITLE
`graph clean` command

### DISCUSSION THREAD

**Discussion by sebastiantf** [graphprotocol/graph-tooling#1360] [Ideas]
It would be really helpful to have a `graph clean` command that removes all generated and build artifacts to get a clean state. This feature is something that's offered by most other Ethereum/blockchain dev tools like Hardhat, Foundry, etc.

**Comment by azf20** [comment]
This makes sense to me! @saihaj what do you think?

**Reply by saihaj** [reply]
Great idea, created a task for adding support for it https://github.com/graphprotocol/graph-tooling/issues/1361

**Comment by saihaj** [comment]
Available as of [v0.51.0](https://github.com/graphprotocol/graph-tooling/releases/tag/%40graphprotocol%2Fgraph-cli%400.51.0)
